google-cloud-bigquery-storage-v1/lib/google/cloud/bigquery/storage/v1/big_query_read.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Auto-generated by gapic-generator-ruby. DO NOT EDIT!
+
+require "google/cloud/bigquery/storage/v1/big_query_read/client"
+require "google/cloud/bigquery/storage/v1/big_query_read/credentials"
```

Comment:
There's a `Bigquery` vs `BigQuery` issue here. It looks like the general branding preference is for `BigQuery`, which matches what other languages are doing. We may have to do something similar to what we did for PubSub, and alias our module names.

Reply:
Will this always be a manual modification?

Reply:
Or can we add configurable aliasing to the generator?

Reply:
Not sure. We'll have to figure it out. The microgenerator provides "helper.rb" hooks, so we have the ability to add in handwritten aliases.

Reply:
But yeah, you're right, we could also update the generator to support configurable aliasing. I think it would be worth handwriting what we think it should look like in this PR first, and then we can move it to the generator later.

Reply:
The other inconsistency is in the path: `bigquery/...` vs `.../big_query...`. But I think we've already decided not to worry about that in the generator, correct?

Reply:
@dazuma As you pointed out offline, both [google-cloud-bigquery](https://github.com/googleapis/google-cloud-ruby/tree/master/google-cloud-bigquery) and [google-cloud-bigquery-data_transfer](https://github.com/googleapis/google-cloud-ruby/tree/master/google-cloud-bigquery-data_transfer) use the `Bigquery` module name instead of `BigQuery`.

Reply:
Yes, we decided offline to stay with the current capitalization for now. We might choose to "fix" it in the future, but when we do so, we should do all three bigquery libraries at once, and leave an alias in the original name.